From 69d5b290a515f6a4c675636dccef4d809a0f66f9 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:51:15 -0400
Subject: [tools] lib/traceevent: Remove print_app_build variable

Message-id: <1408953210-26343-188-git-send-email-jolsa@redhat.com>
Patchwork-id: 88974
O-Subject: [PATCH RHEL7.1 BZ1133083 187/322] tools lib traceevent: Remove print_app_build variable
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit db72a330587e10d6b8009e0dcf3d97aa9b1e2659
Author: Jiri Olsa <jolsa@redhat.com>
Date: Thu Dec 19 14:42:01 2013 +0100

description
===========
Removing print_app_build variable, because it's not needed.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/lib/traceevent/Makefile b/tools/lib/traceevent/Makefile
index 0d9cbb4..022c987 100644
--- a/tools/lib/traceevent/Makefile
+++ b/tools/lib/traceevent/Makefile
@@ -151,7 +151,6 @@ override CFLAGS += $(udis86-flags) -D_GNU_SOURCE
 ifeq ($(VERBOSE),1)
   Q =
   print_compile =
-  print_app_build =
   print_fpic_compile =
   print_shared_lib_compile =
   print_plugin_obj_compile =
@@ -160,7 +159,6 @@ ifeq ($(VERBOSE),1)
 else
   Q = @
   print_compile =  echo '  CC       '$(OBJ);
-  print_app_build =  echo '  BUILD    '$(OBJ);
   print_fpic_compile =  echo '  CC FPIC  '$(OBJ);
   print_shared_lib_compile = echo '  BUILD    SHARED LIB '$(OBJ);
   print_plugin_obj_compile = echo '  CC FPIC  '$(OBJ);
@@ -173,10 +171,6 @@ do_fpic_compile =     \
  ($(print_fpic_compile)    \
  $(CC) -c $(CFLAGS) $(EXT) -fPIC $< -o $@)
 
-do_app_build =      \
- ($(print_app_build)    \
- $(CC) $^ -rdynamic -o $@ $(CONFIG_LIBS) $(LIBS))
-
 do_compile_shared_library =   \
  ($(print_shared_lib_compile)  \
  $(CC) --shared $^ -o $@)
-- 
1.7.1